mitx_residential_etl.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+"""
+Script to run SQL queries on mitx residential and upload them to an S3 bucket.
+"""
+
+import csv
+import json
+import os
+import subprocess
+import sys
+from datetime import datetime
+
+try:
+    import requests
+    from logbook import Logger, RotatingFileHandler
+    from sqlalchemy import create_engine
+    from sqlalchemy.sql import text
+except ImportError as err:
+    print("Failed to import module: ", err)
+    sys.exit("Make sure to install logbook, requests and sqlalchemy")
+
+datetime = datetime.now()
+date_suffix = datetime.strftime('%m%d%Y')
+
+# Read settings_file
+try:
+    settings = json.load(open('settings.json'))
+except IOError:
+    sys.exit("[-] Failed to read settings file")
+
+# Configure logbook logging
+logger = RotatingFileHandler(settings['Logs']['logfile'],
+                             max_size=int(settings['Logs']['max_size']),
+                             backup_count=int(settings['Logs']['backup_count']),
+                             level=int(settings['Logs']['level']))
+logger.push_application()
+logger = Logger(__name__)
```

Comment:
Is there a reason that you're overwriting the `logger` variable?

Reply:
No specific reason apart from cookie-cutter kinda code. Removed.